libs/ledger-live-common/src/families/cosmos/js-buildTransaction.ts

```diff
@@ -12,29 +7,56 @@ import {
 import { MsgWithdrawDelegatorReward } from "cosmjs-types/cosmos/distribution/v1beta1/tx";
 import { SignMode } from "cosmjs-types/cosmos/tx/signing/v1beta1/signing";
 import { TxRaw } from "cosmjs-types/cosmos/tx/v1beta1/tx";
-import { defaultCosmosAPI } from "./api/Cosmos";
-import BigNumber from "bignumber.js";
+import type { Account } from "@ledgerhq/types-live";
+import { AminoMsg, AminoSignResponse } from "@cosmjs/amino";
+import {
+  AminoMsgSend,
+  AminoMsgDelegate,
+  AminoMsgUndelegate,
+  AminoMsgBeginRedelegate,
+  AminoMsgWithdrawDelegatorReward,
+} from "@cosmjs/stargate";
+import { cosmos } from "@keplr-wallet/cosmos";
+import { PubKey } from "@keplr-wallet/proto-types/cosmos/crypto/secp256k1/keys";
+import { AuthInfo, Fee } from "@keplr-wallet/proto-types/cosmos/tx/v1beta1/tx";
+import { TxBody } from "cosmjs-types/cosmos/tx/v1beta1/tx";
+import Long from "long";
+import { Coin } from "@keplr-wallet/proto-types/cosmos/base/v1beta1/coin";
+
+type ProtoMsg = {
```

Comment:
What's a proto msg? Is it related to protobuf? So we're handling both amino and protobuf messages at the same time?

Can you please add a couple lines of comments to explain this type?